### ISSUE TITLE
Pager: accessibility issues

### ISSUE THREAD

**Issue by JBismonte** [infor-design/enterprise#4862]
**Describe the bug**
There are two accessibility issues found in the soho-standalone-pager component:
1. The page size button element with aria-haspopup="true" cannot be activated with the Down Arrow or Up Arrow.
2. The soho page buttons appear to use a roving tabindex rather than individually receiving keyboard focus with the tab key. There is no role or visual indication for this functionality. 
3. the tooltip for the first and previous pages disapper after clicking next from the first page

**To Reproduce**
on your local enterprise-ng
1. run npm start
2. Go to http://localhost:4200/ids-enterprise-ng-demo/datagrid-standalone-pager
3. navigate to the pager buttons via keyboard, you cannot navigate from one button to another button via tab, but you can with arrow keys.
4. also when you navigate to the select page size button via keyboard, clicking down doesn't activate the button.
5. also after clicking next, the first and previous button should enable, try hovering your mouse or navigate your keyboard to the first and previous buttons, the tooltip doesnt show up any more.

**Expected behavior**
[VPAT Recommendations]
1. regarding the page size button: Allow Down Arrow and Up Arrow functionality to open the menu. The Down Arrow should function similarly to the Enter or Spacebar. The Up Arrow should open the menu and focus on the last menu item. 
2. regarding the first, previous, next and last buttons: Remove the arrow key functionality and ensure that each of these controls can receive focus with the Tab key instead. 

**Version**
- ids-enterprise-ng: 9.2.0-dev

**Platform**
 - Device (if applicable) :NA
 - OS Version: Windows 10
 - Browser Name: Firefox
 - Browser Version: v85.0

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-34407

**Comment by tmcconechy** [comment]
When we fix this i think we can just make the pager tabbable instead. That would make sense

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.

**Comment by JBismonte** [comment]
Hi Tim, looks like there is an issue that is still not fixed, which is with the standalone pager.
the tooltip for the first and previous pages doesn't appear after clicking next from the first page, this only occurs when on the second page. 

it is still reproducible on ids 4.51.1:
Here is the page where I can reproduce the issue: http://localhost:4200/ids-enterprise-ng-demo/datagrid-standalone-pager

Here is the screen recording:
https://user-images.githubusercontent.com/29673378/119494616-14ff0980-bd94-11eb-8616-39f602ea2173.mp4

**Comment by tmcconechy** [comment]
Yeah lets make a separate issue. We decided to leave that part as we had three people look at it and no one could figure it out

**Comment by JBismonte** [comment]
Ok, Thanks Tim, created this ticket #5228
